projects/components/src/table/controls/table-controls.component.scss

```diff
@@ -15,7 +15,7 @@
     display: flex;
 
     .search-box {
-      width: 256px;
+      min-width: 256px;
```

Comment:
Why min width here? Do we want it to grow on a table?

Reply:
I noticed the search box shrank in width, I believe from some parent flex-boxing that went in. Changing to min-width fixed the issue.